nncf/hw_configs/vpu.json

```diff
@@ -2,55 +2,60 @@
     "target_device": "VPU",
     "config": {
         "quantization": {
-            "q8_sym_tn": {
+            "q8_tn": {
```

Comment:
Can we explicitly add `asym` here? From this naming, it can be not obvious why q4_tn is symmetric, but q8_tn - asymmetric.